egasub/utils.py

```diff
@@ -137,7 +135,7 @@ def ftp_files(path, ctx):
     files = []
     try:
         files = ftp.nlst(path)
-    except ftplib.error_perm, resp:
+    except ftplib.error_perm:
         echo('Error: unable to connect to FTP server.', err=True)
         ctx.abort()
```

Comment:
this function is not actually used, it was a result of copy-and-paste from previous code, remove this function `ftp_files`